client/src/component/ScrollLoader.tsx

```diff
@@ -0,0 +1,8 @@
+import ReactLoading from "react-loading";
+export default function ScrollLoader() {
+  return (
+    <div className="flex items-center justify-center w-full mb-10 text-center h-3/4">
+      <ReactLoading type="spin" color="black"></ReactLoading>
```

Comment:
```suggestion
      <ReactLoading type="spin" color="black" />
```

저는 요런걸 선호하는데 한번 왜 이래야하는지 찾아봐야할것 같긴하네요 ㅎㅎ